pyproject.toml

```diff
@@ -188,3 +192,37 @@ version = { attr = "nemo_curator.package_info.__version__" }
 [tool.setuptools.packages.find]
 include = ["*"]
 exclude = ["tests", "tests.*"]
+
+[tool.ruff]
+line-length = 88
```

Comment:
idk if folks have other opinions but we can use this switch to also update some of these settings to match the rest of the Nemo framework.